src/components/ec-checkbox/ec-checkbox.story.js

```diff
@@ -137,28 +85,75 @@ stories
         <h3>Error - with multiline label text</h3>
         <ec-checkbox
           :error-message="errorMessageMultiline"
-          class="ec-mb--24 ec-col-6">
+          class="tw-mb-24 tw-col-12">
           <template #label>
             Lorem ipsum dolor sit amet consectetur adipisicing elit. Provident eos consequatur quas reiciendis aliquid ipsam ea pariatur dolorem, molestias maiores.
           </template>
         </ec-checkbox>
 
         <h3>Disabled</h3>
-        <ec-checkbox class="ec-mb--24 ec-col-6"
+        <ec-checkbox class="tw-mb-24 tw-col-12"
           disabled>
           <template #label>
             I accept the <a href="#" @click.stop.prevent="$emit('OpenTerms')"> terms and conditions </a>
           </template>
         </ec-checkbox>
 
         <h3>Disabled - checked</h3>
-        <ec-checkbox class="ec-mb--24 ec-col-6"
+        <ec-checkbox class="tw-mb-24 tw-col-12"
           checked
           disabled>
           <template #label>
             I accept the <a href="#" @click.stop.prevent="$emit('OpenTerms')"> terms and conditions </a>
           </template>
         </ec-checkbox>
+
+        <h3> Label and Error messages coming from props</h3>
+        <ec-checkbox
+          v-if="!valueFromPropsHasError"
+          class="tw-mb-24"
+          v-model="checkbox1"
+          :disabled="valueFromPropsDisabled"
+          :label="valueFromPropsLabel">
+        </ec-checkbox>
+        <ec-checkbox
+          v-else="valueFromPropsHasError"
+          class="tw-mb-24"
+          v-model="checkbox1"
+          :disabled="valueFromPropsDisabled"
+          :label="valueFromPropsLabel"
+          :error-message="valueFromPropsErrorMessage">
+        </ec-checkbox>
+
+        <h3> Label and Error messages coming from template</h3>
+        <ec-checkbox
+          v-if="!valueFromPropsHasError"
+          v-model="checkbox2"
+          :disabled="valueFromPropsDisabled">
+          <template #label>
+            I accept the <a href="#" @click.stop.prevent="$emit('OpenTerms')"> terms and conditions </a>
```

Comment:
It seems like this slips through the previous reviews: it should follow our conventions: `open-terms`.

Reply:
Done